src/scene_title.h

```diff
@@ -103,17 +119,59 @@ class Scene_Title : public Scene {
 private:
 	void OnTitleSpriteReady(FileRequestResult* result);
 
+	/**
+	 * Moves a window (typically the New/Continue/Quit menu) to the middle or bottom-center of the screen.
+	 * @param centerVertical If true, the menu will be centered vertically. Otherwise, it will be at the bottom of the screen.
```

Comment:
lacks ``@window`` documentation.
Also in general your naming convention again: ``centerVertical -> center_vertical``

Reply:
Fixed missing documentation.
Fixed all the camel case names I could find.